hyperscale/Dockerfile

```diff
@@ -0,0 +1,58 @@
+# This file is auto generated from it's template,
+# see citusdata/tools/packaging_automation/templates/docker/latest/latest.tmpl.dockerfile.
+FROM postgres:14.0
+ARG VERSION=10.2.2
+LABEL maintainer="Citus Data https://citusdata.com" \
+      org.label-schema.name="Citus" \
+      org.label-schema.description="Scalable PostgreSQL for multi-tenant and real-time workloads" \
+      org.label-schema.url="https://www.citusdata.com" \
+      org.label-schema.vcs-url="https://github.com/citusdata/citus" \
+      org.label-schema.vendor="Citus Data, Inc." \
+      org.label-schema.version=${VERSION} \
+      org.label-schema.schema-version="1.0"
+
+ENV CITUS_VERSION ${VERSION}.citus-1
+
+# install Citus
+RUN apt-get update \
+    && apt-get install -y --no-install-recommends \
+       ca-certificates \
+       curl \
+    && curl -s https://install.citusdata.com/community/deb.sh | bash \
+    && apt-get install -y postgresql-$PG_MAJOR-citus-10.2=$CITUS_VERSION \
+                          postgresql-$PG_MAJOR-hll=2.16.citus-1 \
+                          postgresql-$PG_MAJOR-topn=2.4.0 \
+    && apt-get purge -y --auto-remove curl \
+    && rm -rf /var/lib/apt/lists/*
+
+# install other extensions
+RUN     apt-get update \
+      &&apt-get install -y postgresql-14-postgis-3 \
+            postgresql-plperl-14 \
+            postgresql-contrib \
+            postgresql-14-partman \
+            postgresql-14-tdigest \
+            postgresql-14-mysql-fdw
+
+RUN apt-get -y install postgresql-14-cron
+
+# add citus to default PostgreSQL config
+RUN echo "shared_preload_libraries='citus,pg_cron,pg_stat_statements'" >> /usr/share/postgresql/postgresql.conf.sample
+
+# add pg_cron into default PostgreSQL config
+RUN echo "cron.database_name = 'postgres'" >> /usr/share/postgresql/postgresql.conf.sample
```

Comment:
I created a citus database but I couldn't make pg_cron work with the citus database. I got the below error
```
2021-11-08 10:35:59.191 UTC [72] STATEMENT:  CREATE EXTENSION pg_cron;
psql:/docker-entrypoint-initdb.d/005-other-extensions.sql:9: ERROR:  can only create extension in database citus
DETAIL:  Jobs must be scheduled from the database configured in cron.database_name, since the pg_cron background worker reads job description
s from this database.
HINT:  Add cron.database_name = 'postgres' in postgresql.conf to use the current database.
CONTEXT:  PL/pgSQL function inline_code_block line 4 at RAISE
```

Reply:
That probably means we are not creating the extensions in the citus database. 

Is there a way we could execute the SQL files below in the `citus` database?

Reply:
It seems we should be able to set the POSTGRES_DB environment variable
https://github.com/docker-library/postgres/blob/dd84d6fccbab39bb67fb8f80b9fb2bfc5cb0c992/docker-entrypoint.sh#L191

Reply:
Fixed